mordred/SP3A2C.py

```diff
@@ -0,0 +1,43 @@
+# Based on the code from https://github.com/rdkit/rdkit/blob/master/Contrib/PBF/pbf.py
+
+from ._base import Descriptor
+
+__all__ = (
+    "SP3A2C",
+)
+
+
+class SP3A2C(Descriptor):
+    r"""SP3A2C descriptor.
+    """
+
```

Comment:
please add `__slots__ = ()` attribute

Reply:
and add `since = "1.1.0"` attribute, please (it is used preset versionning in current develop branch).